misc/atrium-spec/src/main/kotlin/ch/tutteli/atrium/spec/integration/MapAssertionsSpec.kt

```diff
@@ -41,13 +45,27 @@ abstract class MapAssertionsSpec(
     val expect = verbs::checkException
     val fluent = assert(mapOf("a" to 1, "b" to 2))
 
+    val (containsKey, containsKeyFun) = containsKeyPair
     val (hasSize, hasSizeFun) = hasSizePair
     val (isEmpty, isEmptyFun) = isEmptyPair
     val (isNotEmpty, isNotEmptyFun) = isNotEmptyPair
 
     val isDescr = DescriptionBasic.IS.getDefault()
     val isNotDescr = DescriptionBasic.IS_NOT.getDefault()
     val empty = DescriptionCollectionAssertion.EMPTY.getDefault()
+    val containsKeyDescr = DescriptionMapAssertion.MAP_CONTAINS.getDefault()
+
+    describeFun(containsKey) {
+        it("does not throw if a map contains key a") {
+            assert(mapOf()).containsKeyFun("a")
+        }
+
+        it("throws an AssertionError if map not contains key") {
+            expect {
+                assert(mapOf( "b" to 2)).containsKeyFun("a")
+            }.toThrow<AssertionError> { messageContains("$isDescr: $containsKeyDescr")}
```

Comment:
We want to check two things here:
- does the message contain the assertion (could fail for another reason)
- does the assertion include your description (`$containsKeyDescr...`) as well as the representation of the key
additional hint, no need for `$isDescr` here